File: spark-streaming-scala-app/src/test/scala/com/sapient/learning/UtilityUnitTest.scala
package com.sapient.learning

import org.scalatest.FunSuite

class UtilityUnitTest extends FunSuite {

  test("testUtility") {

    var str = "RT @Femi_Sorry: I am sick to death of privileged politicians saying that they personally would be happy with No Deal Brexit."

    println(Utility.cleanse(str).length)
    assert(Utility.cleanse(str).length === 9)
  }

}

File: spark-streaming-java-app/src/test/java/com/sapient/learning/UtilityUnitTest.java
package com.sapient.learning;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;

public class UtilityUnitTest {
	
	@Test
	public void testUtility() {
		
		String str = "RT @Femi_Sorry: I am sick to death of privileged politicians saying that they personally would be happy with No Deal Brexit.";
		
		String[] words = Utility.cleanse(str);
		
		System.out.println(words.length);
		
		assertEquals(9, words.length);;
		
	}

}
